src/PaymentGateways/Gateways/PayPalCommerce/PayPalCommerceGateway.php

```diff
@@ -60,14 +64,15 @@ public function formSettings(int $formId): array
             'donationFormId' => $formId,
             'donationFormNonce' => wp_create_nonce("give_donation_form_nonce_{$formId}"),
             'sdkOptions' => $this->getPayPalSDKOptions($formId),
+            'validateUrl' => (new GenerateDonationFormValidationRouteUrl())(),
```

Comment:
Since we aren't passing constructor arguments we can omit the constructor syntax and remove the nested parenthesis.

```suggestion
            'validateUrl' => (new GenerateDonationFormValidationRouteUrl)(),
```

Reply:
Heads up @kjohnson what you are suggesting would not follow PSR-12 standards which is what we use on the Give Team. I've seen this omitting of parenthesis before and noticed the php storm warning about PSR12.

https://www.php-fig.org/psr/psr-12/#4-classes-properties-and-methods

>When instantiating a new class, parentheses MUST always be present even when there are no arguments passed to the constructor.

Reply:
Ah, ok - that works for me!